Index: app/src/main/java/com/alxbyd/cashcare/test/Test.kt
===================================================================
diff --git a/app/src/main/java/com/alxbyd/cashcare/test/Test.kt b/app/src/main/java/com/alxbyd/cashcare/test/Test.kt
deleted file mode 100644
--- a/app/src/main/java/com/alxbyd/cashcare/test/Test.kt	(revision 65b61d12813b45847228f33e3cb36e8d7b9b8c63)
+++ /dev/null	(revision 65b61d12813b45847228f33e3cb36e8d7b9b8c63)
@@ -1,4 +0,0 @@
-package com.alxbyd.cashcare.test
-
-class Test {
-}
\ No newline at end of file
